tests/global_config.py

```diff
@@ -136,6 +136,16 @@ def _get_default_storage_class(sc_list):
             "default": True,
         }
     },
+    {
+        StorageClassNames.GPFS: {
+            "volume_mode": DataVolume.VolumeMode.FILE,
+            "access_mode": DataVolume.AccessMode.RWX,
+            "snapshot": True,
+            "online_resize": True,
+            "wffc": False,
+            "default": False,
+        }
+    },
```

Comment:
do we really need to have it in default global_config? 
may be we should have separate file for it? like we do here for example:

https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/tests/global_config_ibm_spectrum_sc.py


cc @jpeimer

Reply:
Yeah, I just noticed that we already have the **global_config_ibm_spectrum_sc.py**, so it would be better to use it.

Reply:
+1